src/traces/contour/style.js

```diff
@@ -31,9 +31,13 @@ module.exports = function style(gd) {
         var line = trace.line;
         var cs = contours.size || 1;
         var start = contours.start;
-        var colorLines = contours.coloring === 'lines';
 
-        var colorMap = makeColorMap(trace);
+        // for contourcarpet only - is this a constraint-type contour trace?
```

Comment:
A bit funny to have `contourcarpet only` code in `contour.style` - but perhaps someday we would like to support constraint-type coloring for regular contour plots too?